scripts/install_karavi_auth.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+#
+# Copyright (c) 2022 Dell Inc., or its subsidiaries. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+
+function usage() {
+  echo
+  echo "Help for $0"
+  echo
+  echo "Usage: $0 mode options..."
+  echo "Mode:"
+  echo "  --upgrade                                                   Upgrades CSM Authorization when CSM Authorization is already installed"
+
+  echo
+  echo "  Optional"
+  echo "  --help                                                      Help"
+  echo
+
+  exit 0
+}
+
+UPGRADE=0
+
+while getopts ":h-:" optchar; do
+  case "${optchar}" in
+  -)
+    case "${OPTARG}" in
+    upgrade)
+      UPGRADE=1
+      ;;
+    help)
+      usage
+      exit 0
+      ;;
+    *)
+      echo "Unknown option --${OPTARG}"
+      echo "For help, run $PROG -h"
+      exit 1
+      ;;
+    esac
+    ;;
+  *)
+    echo "Unknown option -${OPTARG}"
+    echo "For help, run $PROG -h"
+    exit 1
+    ;;
+  esac
+done
+
+if [ $UPGRADE == 1 ]; then
+    rpm -Uvh karavi-authorization-1.4-0.x86_64.rpm --nopreun --nopostun
```

Comment:
with the version hardcoded, is this script only supposed to work with 1.4.0?

Reply:
No this will have to be updated for future versions. I can include a version variable on top of this script.

Reply:
Done